djconnectwise/tests/test_sync.py

```diff
@@ -97,8 +98,8 @@ def test_sync_update(self):
         self._sync(new_json_list)
 
         changed = self.model_class.objects.get(id=instance_id)
-        self.assertNotEqual(original.name,
-                            name)
+
+        self.assertNotEqual(getattr(original, self.name), name)
```

Comment:
Follow the pattern of `OpportunityTypes` (only one example but there are MANY others) and override `test_sync_update` instead. 

Its good to test fields unique to those objects and keep to the patterns we already have, BUT I will say that this is a nice solution :)